services/load-certs.js

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
consider your repo as a module.

```json5
{
  "dependencies": {
    /* ... */,
    "node-load-cert-dir": "github:throwaway96/node-load-cert-dir"
  }
}
```

see https://stackoverflow.com/a/45156653.

Reply:
You think the separate module is cleaner? There are advantages, I guess...

Reply:
If you do it this way, it should be pinned to a specific commit. Part of me feels like it's better off just living in this repo, but it could be useful to other WebOS apps.

Reply:
> You think the separate module is cleaner? There are advantages, I guess...

well, if you want to make changes, then you need to manually keep the repository and module in sync.

Reply:
> If you do it this way, it should be pinned to a specific commit. Part of me feels like it's better off just living in this repo, but it could be useful to other WebOS apps. 

you can also use tags. `latest` is quite suitable. ;P

Reply:
Done.